examples/sycl/pvc/flash_attention_v2/pvc_flash_attn_gemm_universal.hpp

```diff
@@ -292,51 +292,47 @@ class GemmUniversalAttention
-      Tensor prefetch_iter_k = append_pvc_tensor< CollectiveMainloop::is_k_transposed ? 1 : 0>(prefetch_iter_ndim, iter_over_head_count, BLK_N);
-
-    // V is a transposed matrix, So here the Sequense length is consumed, it is transposed so the consumed dimension looks like B matrix
-    // Hence, the Head size is the fast moving dimention and horisontal and sequence length is vertical.
-    // The prefetch only move along the sequence lenth. Here we call sequence length K since it get consumed and head size N since it stay
-    // subgroup arranged 4x2 to load (32x64) in one load(each 8x32)
-    Tensor prefetch_iter_2d_v = params.mainloop.gmem_prefetch_v.get_pvc_tensor(
-         make_coord((sub_group_id / ATOM_N) * get<0>(PrefetchVTileSize{}), // iteration 0/K/Hight/vertical/ sequence lengh
-                    head_size_coord,         //  iteration 1/N/W/Horisontal / Head size
-                    blk_l_coord),
-          // We loop over the consuming dimension which is the iteration 0(N) here 
-         make_shape(_1{}, _1{}, _1{}));
-         // first one is to use the intrinsic along the vertical , Second one is N/M  and third one is K
-    Tensor prefetch_iter_v = append_pvc_tensor<0>(prefetch_iter_2d_v, nblock_limit, BLK_K);
+      Tensor prefetch_iter_k = append_pvc_tensor<1>(prefetch_iter_ndim, iter_over_head_count, (get<1>(PrefetchKThrShape{}) * get<1>(PrefetchKTileSize{})));
+
+      // V is a transposed matrix, So here the Sequense length is consumed, it is transposed so the consumed dimension looks like B matrix
+      // Hence, the Head size is the fast moving dimention and horisontal and sequence length is vertical.
+      // The prefetch only move along the sequence lenth. Here we call sequence length K since it get consumed and head size N since it stay
+      // subgroup arranged 4x2 to load (64x64) in one load(each 64x32)
+      Tensor prefetch_iter_2d_v = params.mainloop.gmem_prefetch_v.get_pvc_tensor(
+          make_coord((sub_group_id / get<1>(PrefetchVThrShape{})) * get<0>(PrefetchVTileSize{}),                       // iteration 0/K/Hight/vertical/ sequence lengh
+                     BlockIdxX() * BLK_N + ((sub_group_id % get<1>(PrefetchVThrShape{})) * get<1>(PrefetchVTileSize{})), //  iteration 1/N/W/Horisontal / Head size
+                     blk_l_coord),
+          // We loop over the consuming dimension which is the iteration 0(N) here
+          make_shape(_1{}, _1{}, _1{}));
+      Tensor prefetch_iter_v = append_pvc_tensor<0>(prefetch_iter_2d_v, nblock_limit, (get<0>(PrefetchVThrShape{}) * get<0>(PrefetchVTileSize{})));
     CUTLASS_PRAGMA_UNROLL
     for (int i = 0; i < k_tile_count; i++) {
         prefetch(params.mainloop.gmem_prefetch_q, prefetch_iter_q(_, _, _, i));
     }
-    auto Prefetch_per_workgroup = cute::min(nblock_limit, DispatchPolicy::Stages);
+    const auto Prefetch_per_workgroup = DispatchPolicy::Stages; // cute::min(nblock_limit, DispatchPolicy::Stages);
```

Comment:
WIP?

Reply:
Sorry removed the commented part as this is not needed anymore with the change of the prefetch